### PR TITLE
Jest: Test Product Description

### PR DESCRIPTION
**Pull request by ardelato**
## Description

These tests cover the requirements for the product description component as listed https://github.com/ifixit/react-commerce/issues/1042.

## CR Notes:
There were some additional changes that needed to be made to the `renderWithAppContext` function as we required the context of the `ChakraProvider` as well. I opted to use the `AppProviders` function we use in our main app to render the components for testing.

## QA Notes:
- [x] Confirm CI passes

Closes: #1042

### CODE REVIEW

**Comment by jordycosta** [comment]
QA 🟢 

CI is passing 💯 

Screenshot 2022-12-08 083416

File: frontend/test/jest/utils.tsx [hunk 1]
```diff
@@ -1,12 +1,31 @@
 import { render, RenderOptions, RenderResult } from '@testing-library/react';
-import { AppProvider } from '@ifixit/app';
+import { AppProviders } from '@components/common';
```
Comment (batbattur) [inline frontend/test/jest/utils.tsx:2]: What's the difference between `{ AppProvider } from '@ifixit/app'` and `{ AppProviders } from '@components/common'`?

Comment (batbattur) [reply frontend/test/jest/utils.tsx:2]: Oh my bad, just read the commit message.

**Review by batbattur — APPROVED** [review]
CR 🚛

**Pull request merged**